rbtcwrapperproxy/RBTCWrapperProxy.sol

```diff
@@ -321,7 +320,8 @@ contract RBTCWrapperProxy{
 
             if (address(reserveToken) == wrbtcTokenAddress) {
                 IWrbtcERC20(wrbtcTokenAddress).withdraw(reserveAmount);
-                msg.sender.transfer(reserveAmount);
+                (bool successOfSendRBTC,) = msg.sender.call.value(reserveAmount)("");
+                require(successOfSendRBTC, "Failed to send RBTC to user");
             } else {
                 IERC20Token(reserveToken).transfer(msg.sender, reserveAmount);
```

Comment:
you must check return value of `ERC20.transfer`, especially for an arbitrary token